packages/web/src/common/store/backend/sagas.ts

```diff
@@ -47,34 +33,7 @@ export function* waitForBackendSetup() {
   }
 }
 
-function* awaitReachability() {
```

Comment:
just wondering where this went, since guessing this would be around nikki's work?

Reply:
Ah this is my mistake, I deleted the RequestNetworkConnected interface and thought this could all be removed. But i'll put it back to minimize scope of this pr!